src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Enhancing Node-RED with High Availability in FlowForge 1.8
+subtitle: Empowering Node-RED with increased availability and streamlined deployment pipelines
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+#image: /blog/2023/05/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
+
+<!--more-->
+
+## High Availability
+Are you running business-critical processes in Node-RED and need to ensure they're always available and can handle increasing workloads? We heard you! We're excited to introduce High Availability (HA) in FlowForge to support your needs.
+
+Say you're managing a large influx of HTTP requests to your Node-RED instances. With our latest update, FlowForge now ensures that these requests are distributed evenly across your instances. This means your applications are less likely to experience downtime and can handle higher traffic loads​.
```

Comment:
Isn't the main point that we are creating two instances to run the same flow and a load balancer will distribute the load? I think you should explain this fact. Just sayin distributed evenly across your instances leaves a lot of unanswered questions, imho.